### DISCUSSION TITLE
Allow advanced customization when entering details manually

### DISCUSSION THREAD

**Discussion by contextnerror** [ente-io/ente#1031] [Enhancements]
Based on issue #109, it seems like it is possible to have advanced token customization (e.g. not  6 digits). However there is no way to set this when manually making an entry.

Please allow manual configuration of advanced settings (totp/hotp, algorithm, digits, refresh time/initial counter, etc.) when creating a new entry.

**Comment by contextnerror** [comment]
Any word on this?

**Comment by ameknite** [comment]
I want the same, It seems it works with QR codes, but it's annoying using your phone when the app should work by itself in desktop

**Comment by ua741** [comment]
Are there any popular websites that use different algorithm or digits?

**Comment by contextnerror** [comment]
Anything that uses Authy was able to be imported into other 2fa apps at the time of issue creation, and those used a nonstandard number of digits.

Steam uses 5 digits.
Battle.net uses 8 digits.
Sophos firewall can use other algorithms.
Australia mygov uses sha512.
privacyIDEA based 2fa can use other algorithms.

I’m sure there are others.
